data/supporters.json

```diff
@@ -1647,6 +1647,44 @@
           "email": "j.a.todorova@gmail.com"
         }
       ]
-    }
+    },
+    {
+      "name": "FOSS Backstage",
+      "city": "Berlin",
+      "country": "Germany",
+      "link": "https://foss-backstage.de",
+      "twitter": "fossbckstg",
+      "contacts": [
+        {
+          "name": "Ekaterina Klykova",
+          "email": "ekl@newthinking.de"
+          "phone":   "+493092105978"
```

Comment:
There are several parsing errors in the JSON files. You can see the red markers here on Github. Please fix first. Thanks :)